src/pages/workshop-description/index.js

```diff
@@ -122,7 +122,7 @@ export const WorkshopDetails = () => {
               <li>
                 <div className="div-local">
                   <p className="local">
-                    <div className="spacingLocal">
+                    <div className="spacing-local">
```

Comment:
Em inglês, `local` pode acabar sendo um pouco ambíguo no contexto de localização. Sugestões:

```suggestion
                    <div className="spacing-venue">
```

```suggestion
                    <div className="spacing-location">
```